packages/ui/src/accounts/model/lockTypes.ts

```diff
@@ -57,7 +57,13 @@ const isConflictingWith = (lockTypeA: LockType): ((lockTypeB: LockType) => boole
   }
 }
 
-export const isRecoverable = (type: LockType): boolean => RECOVERABLE.includes(type)
+export const isRecoverable = (type: LockType, isActiveCandidate?: boolean): boolean => {
+  if (isActiveCandidate) {
+    return type === 'Voting'
+  }
+
+  return RECOVERABLE.includes(type)
```

Comment:
It relies on `RECOVERABLE` only having 2 lock types. This would be more future proof and IMO clearer:
```suggestion
  if (!RECOVERABLE.includes(type)) {
    return false
  }
  
  switch (type) {
    case 'Council Candidate':
      return !isActiveCandidate
    
    default:
      return true
  }
```